docs/core/deploying/creating-nuget-packages.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Create a NuGet package with .NET Core CLI
+title: Create a NuGet package with CLI
```

Comment:
There is something wrong with this article. I can't find it by searching for the exact title in Google. Is there something you suggest to change in the metadata to improve its discoverability ?

Reply:
In Google I couldn't find it, even went through until the 6th page without finding it. Even if NuGet articles has higher ranks, it should at least appear after them.

Reply:
The article needs a rewrite. There is a note at the start of the article, which doesn't help SEO. There is only a single H2 which is a statement and doesn't really have any good keywords. The description metadata is short too.